src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/service/spreadsheet/outbound/PriceSheet.kt

```diff
@@ -1,6 +1,6 @@
 package uk.gov.justice.digital.hmpps.pecs.jpc.service.spreadsheet.outbound
 
-import org.apache.pdfbox.util.Hex
+import org.apache.commons.codec.binary.Hex
```

Comment:
Moving to a later version of Apache POI this was no longer available in this package hence the change.